src/components/tab-group/tab-group.scss

```diff
@@ -0,0 +1,61 @@
+@import 'variables';
+@import 'default-theme';
+
+:host {
+  display: block;
+  font-family: sans-serif;
+  .md-tab-header {
+    border-bottom: 1px solid md-color($md-background, status-bar);
+  }
+}
+
+.md-tab-header {
+  overflow: hidden;
+  position: relative;
+  display: block;
+  &:before, &:after {
+    content: '';
+    clear: both;
+    display: table;
+  }
+  &:focus {
+    outline: none;
+    .md-tab-label.md-active {
+      background-color: md-color($md-primary, 100, 0.2);
+    }
+  }
+}
+
+.md-tab-label {
+  float: left;
+  line-height: 48px;
+  height: 48px;
```

Comment:
Have you considered putting this into a Sass variable?  Might help make it more customizable.